examples/bert/bert_train.py

```diff
@@ -53,6 +58,30 @@
     "Skip restoring from checkpoint if True",
 )
 
+flags.DEFINE_bool(
```

Comment:
What does the full invocation end up looking like? Could we make this more simple where all of these paths could be `gs://` paths? Similar to how gfile handles this? So something like

```
bert_train.py \
  --input_files=gs://foo-bucket/foo-bert-data \
  --checkpoint_save_directory=gs://bar-bucket/training-01/pretrained-checkpoints \
  --saved_model_output=gs://bar-bucket/training-01/pretrained-model \
  ...
```

I don't think we should demand that logging and source data are coming from the same bucket. And overall I think a flow like this would be simpler.

Reply:
changed!